adoc/admin-ses-integration.adoc

```diff
@@ -14,623 +14,510 @@ You can check this by running `rpm -q ceph-common` and `rpm -q xfsprogs`.
+      monitors:
+      - '<MONITOR_IP:MONITOR_PORT>' <2> 
+      pool: myPool // <3>
+      image: myVolume // <4>
+      user: myPoolUser // <5>
+      secretRef:
+        name: ceph-secret // <6>
+      fsType: ext4
+      readOnly: false
+====
+<1> The size of the volume image.
+<2> A list of Ceph monitor nodes IP and port. The default port is *6789*.
+<3> The Ceph pool name.
+<4> The Ceph volume image name.
+<5> The Ceph pool user.
+<6> The Kubernetes Secret name contains the Ceph pool user key.
 
-. Create a pod that uses the persistent volume claim:
+. Create a Persistent Volume Claim.
 +
+====
+  kind: PersistentVolumeClaim
+  apiVersion: v1
+  metadata:
+    name: sample-rbd
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi // <1>
+====
+<1> The request volume size in _gibibit_ notation.
+[[NOTE]]
+====
+When you delete the pod, the persistent volume claim is deleted as well.
+The RBD is not deleted.
+====
 
-----
-kubectl apply -f - <<*EOF*
-apiVersion: v1
-kind: Pod
-metadata:
-  name: <POD_NAME>
-spec:
-  containers:
-  - name: <CONTAINER_NAME>
-    image: <IMAGE_NAME>
-    volumeMounts:
-    - mountPath: /mnt/rbdvol
-      name: rbdvol
-  volumes:
-  - name: rbdvol
-    persistentVolumeClaim:
-      claimName: <PV_NAME>
-*EOF*
-----
-. Verify that the pod exists and its status:
+. Create the Pod:
 +
+====
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: sample-rbd
+  spec:
+    containers:
+    - name: sample-container
+      image: busybox
+      volumeMounts:
+      - mountPath: /mnt/rbdvol // <1>
+        name: rbdvol
+    volumes:
+    - name: rbdvol
+      persistentVolumeClaim:
+        claimName: sample-rbd // <2>
+====
+<1> The volume mount path inside the Pod.
+<2> The Persistent Volume Claim name.
 
-----
-kubectl get pod
-----
-. Once the pod is running, check the volume:
+. Once the pod is running, check the volume is mounted:
 +
-
 ----
-kubectl exec -it CONTAINER_NAME -- df -k ...
+kubectl exec -it sample-container -- df -k ...
 /dev/rbd3               999320      1284    929224   0% /mnt/rbdvol
 ...
 ----
 
 
-In case you need to delete the pod, run the following command:
+[#RBD-dynamic-persistent-volumes]
+=== Using RBD in Storage Classes
+
+
+The following procedure describes how use RBD in Storage Class:
+
+. link:https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool[Create a Ceph pool].
 
+. Create pool admin in the Ceph cluster, and record the key:
++
 ----
-kubectl delete pod <CONTAINER_NAME>
+ceph auth get-or-create-key client.myAdmin mds 'allow *' mgr 'allow *' mon 'allow *' osd 'allow * pool=myPool'  | tr -d '\n' | base64
 ----
-
-.Deleting A Pod
 [NOTE]
 ====
-When you delete the pod, the persistent volume claim is deleted as well.
-The RBD is not deleted.
+To improve authentication security, go to link:https://documentation.suse.com/ses/7/single-html/ses-admin/#strong-authentication[Strong authentication] for more information.
```

Comment:
This is referencing to https://github.com/SUSE/doc-ses/blob/master/xml/hardening_measures_authentication.xml#L21-L75.